patient-portal-ui/src/datajs/DeafutlAllData.tsx

```diff
@@ -1,14 +1,13 @@
 import React, { useState, useEffect } from "react";
 import { dateIsoToString } from '../utils/ManageDate';
-const url_0 = 'http://api-develop.ohpp.local:18080/api/';
-const url_1 = 'http://api-develop.ohpp.local:18080/api/';
-// export const ListDoctorPatientsTest = async () => {
+const url_0 = 'http://localhost:18080/api/';
+
```

Comment:
this is for development, the correct url_0 for production is:
```suggestion
const url_0 = 'http://api-develop.ohpp.local:18080/api/
// const url_0 = 'http://localhost:18080/api/'; //uncomment for development
```